src/llmcompressor/transformers/finetune/README.md

```diff
@@ -74,9 +74,10 @@ train(
 
 Finetuning arguments are split up into 3 groups:
```

Comment:
We should add a new generic README related to this arguments outside from under `transformers`
Could be same location as where the args get moved to, such as `src/llmcompressor/args`